docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+  instances associated with it.
+  In other words, it represent the maximum samples that the middleware can store for a DataReader or DataWriter.
+* **Max instances**: Controls the maximum number of instances that a DataWriter or DataReader can manage.
+* **Max samples per instance**: Controls the maximum number of samples within an instance that the DataWriter or
+  DataReader can manage.
+* **Allocated samples**: States the number of samples currently allocated.
+
+.. note::
+     This QoS Policy concerns to Topic, DataWriter and DataReader entities.
+     :raw-html:`<br />`
+     It cannot be changed after the creation of the entity.
+
+.. _resourcelimits_consistencyrule:
+
+Consistency Rule
+""""""""""""""""
+
+To maintain the consistency within the ResourceLimitsQosPolicy, the values of the data members must follow the next
+conditions:
+
+* The value of `max_samples` must be higher or equal to the value of `max_samples_per_instance`.
+* The value established for the :ref:`historyqospolicy` `depth` must be lower or equal to the value stated for
+  `max_samples_per_instance`.
+
+
+Example
+"""""""
+
+C++
+***
+.. literalinclude:: ../../../../../code/DDSCodeTester.cpp
+   :language: c++
+   :dedent: 8
+   :start-after: //DDS_CHANGE_RESOURCE_LIMITS_QOS_POLICY
+   :end-before: //!
+
+XML
+***
+.. literalinclude:: /../code/XMLTester.xml
+    :language: xml
+    :start-after: <!-->XML_RESOURCE_LIMITS
+    :end-before: <!--><-->
+
+.. _timebasedfilterqospolicy:
+
+TimeBasedFilterQosPolicy
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    This QoS Policy will be implemented in future releases.
+
+Filter that allows a DataReader to specify that it is interested only in a subset of the values of the data.
+This filter states that the DataReader doesn't want to receive more than one value each `minimum_separation`,
+regardless of how fast the changes occur.
+
+The `minimum_separation` must be lower than the :ref:`deadlineqospolicy` `period`.
+By default, the `minimum_separation` is zero, which means that the DataReader is potentially interested in all the
+values.
+
+List of QoS Policy data members:
+
++--------------------------+------------------------------+-------------------+
+| Data Member Name         | Type                         | Default Value     |
++==========================+==============================+===================+
+| minimum_separation       | fastrtps::Duration_t         | c_TimeZero        |
++--------------------------+------------------------------+-------------------+
+
+.. note::
+     This QoS Policy concerns to DataReader entities.
+     :raw-html:`<br />`
+     It can be changed after the creation of the entity.
+
+.. _topicdataqospolicy:
+
+TopicDataQosPolicy
+^^^^^^^^^^^^^^^^^^
+
+Allow the application to attach additional information to a created Topic so that when a remote application discovers
+it can access the data and use it.
```

Comment:
```suggestion
Allow the application to attach additional information to a created Topic so that when it is discovered by a remote application,
it can access the data and use it.
```